lib/ansible/plugins/test/success.yml

```
@@ -5,8 +5,9 @@ DOCUMENTATION:
   short_description: check task success
   aliases: [succeeded, successful]
   description:
-    - Tests if task finished successfully, opposite of C(failed).
+    - Test if task did not end in failure; returning the opposite of C(failed).
```

In the spirit of clarifying overloaded terms, C(failed) should really be a link to the failed test (since it's the test's definition of failed).

```suggestion
    - Test the opposite of P(ansible.builtin.failed#test). This indicates the task has been processed successfully.
```

---

Good idea, I added a link. Thanks!